README.md

```diff
@@ -61,7 +61,7 @@ In this case shutdown of the client is not neccecary.
 
 ## Usage guide
 
-Most common HTTP methods are supported out of the box. In case you need to have more control over the method, or you want to add headers or body, use ```HTTPRequest``` struct:
+Most common HTTP methods are supported out of the box. In case you need to have more control over the method, or you want to add headers or body, use the ```HTTPRequest``` struct:
```

Comment:
single word quoting should be done with `` ` `` and not `` ``` `` because `` ``` `` opens a code-block which in some renderers gets its own line.

Reply:
👍 I copied the style found elsewhere in this doc, I'll fixup 🚫